itou/templates/siaes/show_financial_annexes.html

```diff
@@ -10,79 +10,83 @@ <h1>Annexes financières</h1>
+                                    </a>
+                                </div>
+                            {% endif %}
+                        </div>
+                        <div class="table-responsive-lg mt-3 mt-md-4">
+                            <table class="table">
+                                <caption class="visually-hidden">Liste des annexes financières</caption>
+                                <thead>
                                     <tr>
-                                        <td>Aucune annexe financière associée à cette structure.</td>
-                                        <td></td>
-                                        <td></td>
-                                        <td></td>
+                                        <th scope="col">Numéro d'annexe financière</th>
+                                        <th scope="col">Date de début d'effet</th>
+                                        <th scope="col">Date de fin d'effet</th>
+                                        <th scope="col">Validité à ce jour</th>
                                     </tr>
-                                {% endif %}
-                            </tbody>
-                        </table>
+                                </thead>
+                                <tbody>
+                                    {% if financial_annexes %}
+                                        {% for financial_annex in financial_annexes %}
+                                            <tr>
+                                                <td>{{ financial_annex.number_with_spaces }}</td>
+                                                <td>{{ financial_annex.start_at|date:"d/m/Y" }}</td>
+                                                <td>{{ financial_annex.end_at|date:"d/m/Y" }}</td>
+                                                {% if financial_annex.is_active %}
+                                                    <td>
+                                                        <span class="badge badge-sm rounded-pill bg-success-lighter text-success">Valide</span>
+                                                    </td>
+                                                {% else %}
+                                                    <td>
+                                                        <span class="badge badge-sm rounded-pill bg-primary">Inactive</span>
+                                                    </td>
+                                                {% endif %}
+                                            </tr>
+                                        {% endfor %}
+                                    {% else %}
+                                        <tr>
+                                            <td colspan="4">Aucune annexe financière associée à cette structure.</td>
+                                        </tr>
+                                    {% endif %}
```

Comment:
Car ça me revient en tête, il y a un `for..empty` qui permet d'éviter le `if/for` et une indentation : https://docs.djangoproject.com/en/4.2/ref/templates/builtins/#for-empty

Reply:
Merci @rsebille , c'est mieux en effet ;)